src/lambda/local/configureLocalLambda.ts

```diff
@@ -0,0 +1,245 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import { parse } from 'jsonc-parser'
+import * as os from 'os'
+import * as path from 'path'
+import * as vscode from 'vscode'
+import { accessAsync, mkdirAsync, writeFileAsync } from '../../shared/filesystem'
+import { DefaultSettingsConfiguration } from '../../shared/settingsConfiguration'
+
+export interface HandlerConfig {
+    event: any
+}
+
+export interface HandlersConfig {
+    handlers: {
+        [ handler: string ]: HandlerConfig | undefined
+    }
+}
+
+export async function configureLocalLambda(sourceUri: vscode.Uri, handler: string): Promise<void> {
+    // Handler will be the fully-qualified name, so we also allow '.' despite it being forbidden in handler names.
+    if (/[^\w\-\.]/.test(handler)) {
```

Comment:
What if you only produce the CodeLens if the handler name is valid?

Reply:
✔️